src/node.rs

```diff
@@ -115,6 +122,19 @@ impl Node {
         }
     }
 
+    /// Returns the storage path of the operator.
+    pub(crate) fn operator_storage_path(&self, _local_ip: SocketAddr) -> PathBuf {
```

Comment:
future TODO: it would probably be best to have methods like these implemented in the context of some trait that applies to different types of nodes